### PR TITLE
Stubs Again

### PR DESCRIPTION
**Pull request by emmauss**
Just a couple of stubs.

### CODE REVIEW

File: Ryujinx.HLE/HOS/Services/Aud/IAudioRendererManager.cs [hunk 1]
```diff
@@ -161,13 +163,21 @@ private static int EdgeMatrixGetWorkBufferSize(int Value)
             return Result / 8;
         }
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Aud/IAudioRendererManager.cs]: Don't forget to add method symbol (SwIPC) as comment for consistency. `GetAudioDeviceService(nn::applet::AppletResourceUserId) -> object<nn::audio::detail::IAudioDevice>`

File: Ryujinx.HLE/HOS/Services/Aud/IAudioRendererManager.cs [hunk 2]
```diff
@@ -161,13 +163,21 @@ private static int EdgeMatrixGetWorkBufferSize(int Value)
...
 
-        public long GetAudioDevice(ServiceCtx Context)
+        public long GetAudioDeviceService(ServiceCtx Context)
         {
             long UserId = Context.RequestData.ReadInt64();
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Aud/IAudioRendererManager.cs]: Please rename `UserId` to `AppletResourceUserId` and then add a logging message before `MakeObject` to know what the `AppletResourceUserId` is.

File: Ryujinx.HLE/HOS/Services/Aud/IAudioRendererManager.cs [hunk 3]
```diff
@@ -161,13 +163,21 @@ private static int EdgeMatrixGetWorkBufferSize(int Value)
...
 
             MakeObject(Context, new IAudioDevice(Context.Device.System));
 
             return 0;
         }
+
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Aud/IAudioRendererManager.cs]: Don't forget to add method symbol (SwIPC) as comment for consistency. ` GetAudioDeviceServiceWithRevisionInfo(nn::applet::AppletResourceUserId, u32) -> object<nn::audio::detail::IAudioDevice>`

File: Ryujinx.HLE/HOS/Services/Aud/IAudioRendererManager.cs [hunk 4]
```diff
@@ -161,13 +163,21 @@ private static int EdgeMatrixGetWorkBufferSize(int Value)
...
+        private long GetAudioDeviceServiceWithRevisionInfo(ServiceCtx Context)
+        {
+            Context.Device.Log.PrintStub(LogClass.ServiceAudio, "Stubbed.");
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Aud/IAudioRendererManager.cs]: Please read the `AppletResourceUserId` and the `u32` who is probably the Revision number. Then add a logging message before the `return` to know what those var are. Maybe the Revision here is used for some things... But we can point out that later...

File: Ryujinx.HLE/HOS/Services/Aud/IAudioRendererManager.cs [hunk 5]
```diff
@@ -161,13 +163,21 @@ private static int EdgeMatrixGetWorkBufferSize(int Value)
...
+
+            return GetAudioDeviceService(Context);
+        }
+
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Aud/IAudioRendererManager.cs]: nit: remove extra line

File: Ryujinx.HLE/HOS/Services/Irs/IIrSensorServer.cs [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+        public IIrSensorServer()
+        {
+            m_Commands = new Dictionary<int, ServiceProcessRequest>()
+            {
+                { 302, ActivateIrsensor   },
+                { 303, DeactivateIrsensor }
+            };
+        }
+
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Irs/IIrSensorServer.cs]: Don't forget to add method symbol (SwIPC) as comment for consistency.
`ActivateIrsensor(nn::applet::AppletResourceUserId, pid)`

File: Ryujinx.HLE/HOS/Services/Irs/IIrSensorServer.cs [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+        {
+            long AppletResourceUserId = Context.RequestData.ReadInt64();
+            int  IrsSensorHandle      = Context.RequestData.ReadInt32();
+
+            Context.Device.Log.PrintStub(LogClass.ServiceIrs, $"Stubbed. AppletResourceUserId: {AppletResourceUserId} - " +
+                                                              $"IrsSensorHandle: {IrsSensorHandle}");
+
+            return 0;
+        }
+
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Irs/IIrSensorServer.cs]: Don't forget to add method symbol (SwIPC) as comment for consistency.
`DeactivateIrsensor(nn::applet::AppletResourceUserId, pid)`

File: Ryujinx.HLE/HOS/Services/Irs/IIrSensorServer.cs [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+            {
+                { 302, ActivateIrsensor   },
+                { 303, DeactivateIrsensor }
+            };
+        }
+
+        public long ActivateIrsensor(ServiceCtx Context)
+        {
+            long AppletResourceUserId = Context.RequestData.ReadInt64();
+            int  IrsSensorHandle      = Context.RequestData.ReadInt32();
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Irs/IIrSensorServer.cs]: The `IrsSensorHandle` don't exist here. The `PID-descriptor` is not used inside the emulator.

File: Ryujinx.HLE/HOS/Services/Irs/IIrSensorServer.cs [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+            Context.Device.Log.PrintStub(LogClass.ServiceIrs, $"Stubbed. AppletResourceUserId: {AppletResourceUserId} - " +
+                                                              $"IrsSensorHandle: {IrsSensorHandle}");
+
+            return 0;
+        }
+
+        public long DeactivateIrsensor(ServiceCtx Context)
+        {
+            long AppletResourceUserId = Context.RequestData.ReadInt64();
+            int  IrsSensorHandle      = Context.RequestData.ReadInt32();
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Irs/IIrSensorServer.cs]: The `IrsSensorHandle` don't exist here. The `PID-descriptor` is not used inside the emulator.

File: Ryujinx.HLE/HOS/Services/Mm/IRequest.cs [hunk 10]
```diff
@@ -14,12 +15,20 @@ public IRequest()
...
             {
-                { 4, Initialize },
-                { 6, SetAndWait },
-                { 7, Get        }
+                { 1, InitializeOld },
+                { 4, Initialize    },
+                { 6, SetAndWait    },
+                { 7, Get           }
             };
         }
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Mm/IRequest.cs]: Don't forget to add method symbol (SwIPC) as comment for consistency.
`InitializeOld(u32, u32, u32)`
You can add symbols to other Interface methods if you want too...

File: Ryujinx.HLE/HOS/Services/Mm/IRequest.cs [hunk 11]
```diff
@@ -14,12 +15,20 @@ public IRequest()
...
 
+        public long InitializeOld(ServiceCtx Context)
+        {
+            Context.Device.Log.PrintStub(LogClass.ServiceMm, "Stubbed.");
```
Comment (AcK77) [inline Ryujinx.HLE/HOS/Services/Mm/IRequest.cs]: Can you read all method args and then use a logging to print them.

**Comment by emmauss** [comment]
Addressed comments

**Pull request merged**